src/BlazorWebView/src/Maui/Android/WebKitWebViewClient.cs

```diff
@@ -108,7 +95,7 @@ public override void OnPageFinished(AWebView? view, string? url)
 			base.OnPageFinished(view, url);
 
 			// TODO: How do we know this runs only once?
-			if (view != null && url == AppOrigin)
+			if (view != null && IsAppOriginPageUri(url))
```

Comment:
Is this `TODO` relevant now that due to the reload mechanism this _can_ run more than once?

Reply:
I may be mistaken, but I think this TODO was referring to the case described here: https://github.com/dotnet/maui/blob/29e841af968ec2999fa6f1294169f964d024f4a6/src/BlazorWebView/src/Maui/Android/WebKitWebViewClient.cs#L119-L125

I believe this new change is separate from that scenario.

Reply:
Ah okay, thanks for clarifying!